cpg_utils/hail_batch.py

```diff
@@ -64,6 +72,9 @@ def init_batch(**kwargs):
     kwargs : keyword arguments
         Forwarded directly to `hl.init_batch`.
     """
+
+    billing_project = os.getenv('HAIL_BILLING_PROJECT')
```

Comment:
Looks like this didn't get resolved properly when merging with HEAD?

Reply:
Oh. Yeah, merge artefact. Fixed.

Reply:
There are few more merge artefacts, will address them now